connect/transforms/src/main/java/org/apache/kafka/connect/transforms/TimestampRouter.java

```diff
@@ -29,6 +31,7 @@
 import java.util.regex.Pattern;
 
 public class TimestampRouter<R extends ConnectRecord<R>> implements Transformation<R> {
+    private static final Logger log = LoggerFactory.getLogger(TimestampRouter.class);
```

Comment:
I'm not sure any of these log messages add value since TransformationChain has log messages with the records, which should be sufficient to know whether the schemas are null.